src/Http/Management/UserPermissionsVoter.php

```diff
@@ -69,7 +66,7 @@ public function vote(TokenInterface $token, $object, array $attributes)
                 continue;
             }
 
-            $permission = Permission::fromNative($attribute);
+            $permission = $attribute;
 
             if (!in_array($permission, $userPermissions)) {
```

Comment:
I think the type of `$permission` is `Permission` and the `$userPermissions` is array of `Permission`. So I think `in_array` will always be `false`?

Reply:
+1 for unit test to be sure, but I don't see the problem here immediately to be honest. `in_array` can compare objects in arrays. There's even a third argument to enable/disable strict comparisons so it does or doesn't do type casting. (It type casts by default, but that should be no issue here)

Reply:
I noticed that this code is already covered with a unit test. It indeed works because strict comparison is not enabled.

Reply:
I think it would even work with strict comparison, since it's comparing `Permission` objects on both sides